repository/f.json

```diff
@@ -688,6 +688,18 @@
 				}
 			]
 		},
+		{
+			"name": "FilePrompt",
+			"details": "https://github.com/loquens/subl-open-file-prompt",
+			"labels": ["file navigation", "file manipulation", "utilities", "open files", "file utils", "file suggestions"],
+			"readme": "https://raw.githubusercontent.com/loquens/subl-open-file-prompt/main/README.md",
```

Comment:
You don't need to provide this readme link. Everything is already found via the repo link in "details".

Reply:
Fixed. Seen it in other packages, so decided to add it.